api/src/main/java/org/openmrs/module/ugandaemrreports/data/converter/ARVsDataConverter.java

```diff
@@ -38,10 +38,10 @@ else  if(arvResultsConcept != null && arvResultsConcept.equals(Dictionary.getCon
         }
 
         else  if(arvResultsConcept != null && arvResultsConcept.equals(Dictionary.getConcept("2c000b41-f9d7-40c1-8de0-bce91dbae932"))){
-            return "=CONCATENATE(\"ART\",UNICHAR(8730))";
+            return "ART✔";
```

Comment:
@solemabrothers you need to leave the concatenate otherwise it will not work in Excel